pkgs/os-specific/linux/drbd-utils/default.nix

```diff
@@ -0,0 +1,122 @@
+{ lib, stdenv
+, fetchurl
+, flex
+, systemd
+, perl
+, docbook_xsl
+, libxslt
+, docbook_xml_dtd_44
+, docbook_xml_dtd_45
+, resource-agents
+, kmod
+}:
+
+stdenv.mkDerivation rec {
+  pname = "drbd-utils";
+  version = "9.16.0";
+
+  src = fetchurl {
+    url = "https://linbit.com/downloads/drbd/utils/${pname}-${version}.tar.gz";
+    sha256 = "sha256:0s1dgg4nvvvxzh9qrah3n15b92h38b8m9pa43s14fdjdg11m6wk2";
+  };
+
+  # patches = [ ./pass-force.patch ./fix-glibc-compilation.patch ];
```

Comment:
Whats with this?

Reply:
Those are old patches from the drbd derivation. Removed it.